MT-CTF/capturetheflag
# Increase sword damage dealt by knights

Also makes steel sword craftable again

# How to test
* Join to clients into a CTF game
* Punch a knight with a medic and vise versa. Knight should deal 1 extra damage (Easier to see damage difference when punching medic)
* Craft a steel sword as a non-knight class and use it
* Play around fighting the different classes as a knight
* Try to exploit the damage dealing code
